news/8963.exclude.rst

```diff
@@ -0,0 +1 @@
+Add exclude for ``libwayland.so`` to prevent mismatches with system drivers.
```

Comment:
``libwayland*.so``, to keep the text accurate.

Also, `exclude` is not a valid changelog category. Although, considering that it was not caught anywhere, it might be interpreted as a literal "exclude" by `towncrier`. 

I suppose `bugfix` would be a suitable category.